src/app/app.component.html

```diff
@@ -1,7 +1,11 @@
-<alm-app-header></alm-app-header>
-
-<main class="app">
-  <router-outlet></router-outlet>
+<header class="padding-bottom-10" >
```

Comment:
In semantic html5 a header is a tag with specific meaning from a layout perspective... a separate concern to navigation. The angular alm-app-header is not really a header by function ... it's really a navigation strategy ... The idea is to separate layout (header) from function (navigation) from content  as much as possible. In general this results in more maintainable css and cleaner html5. Navigation could be placed anywhere in the layout depending on different responsive layouts. Also from a styling perspective, separating layout from content always results in fewer css and less nested rules.